pcdet/models/roi_heads/pvrcnn_head.py

```diff
@@ -195,8 +195,7 @@ def forward(self, batch_dict, test_only=False,use_gtboxes=False):
                 if self.model_cfg['NORMALIZATION']:
                     # pooled_features dim : [GT_boxes,27648]
                     pooled_features = F.normalize(pooled_features, dim = -1)
-                if self.model_cfg['PROJECTOR']:
```

Comment:
Can I ask what is projector config that you are ignoring it here? @adv010

Reply:
@frezaeix During initial phases of implementing this loss,  I thought of projection as a good-to-have. However, in almost all contrastive literature, projection is a necessary step to have. Hence, I decided to remove this condition and make it part of the main pipeline.

The normalization is a step that I've picked up when reading the Supervised Contrastive loss paper, where they normalize features at 2 steps :
1.  The features from last layer of the backbone (ResNet)
Since this is new functionality we'll be introducing to the PVRCNN backbone, I chose to make it as an optional setting with the normalize flag.
2. Normalizing the projected embedding 
This is handled  within the contrastive_loss function itself

![MicrosoftTeams-image (2)](https://github.com/fnozarian/OpenPCDet/assets/40948853/8cb67103-516d-4143-91d8-512d63e0b111)
This image shows the mental model of the current pipeline